Documentation/ceph-configuration.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configuration
+weight: 3700
```

Comment:
This topic will come between the CSI and upgrades topics?

Reply:
That was the intent from looking at the current docs on the website. I'm open to other suggestions.